README.md

```diff
@@ -29,14 +29,15 @@ updates is `12345` and for PSP it is `2`.
 
 ## Setup
 
-- Airkeeper will require a configuration file that matches the one being used by the target Airnode. The Airnode's
-  `config.json` and `secrets.env` file can be copied from the Airnode repo and placed in the /config directory of this
-  repo. Examples of these two files can be found in the /config directory of this repo.
+- Airkeeper configuration file will require you to copy over some parts of the Airnode's
+  [`config.json`](https://github.com/api3dao/airnode/blob/v0.8.0/packages/airnode-node/config/config.example.json) file.
```

Comment:
It was because I think the user should be looking at the `config.json` file that matches the version of the airnode dependencies. Otherwise it might add or remove properties in airkeeper.json that are not supported by the airnode.

I know that it will be a hassle to always remember to update this link whenever we update the deps in package.json so should I maybe find a way to extract this value from there? 🤔

Reply:
I don't think it's that important at the moment but feel free to create an issue for that.